www/generate_tutorial/src/tutorial.roc

```diff
@@ -28,7 +28,7 @@ view = \htmlContent ->
                 text htmlContent,
             ],
             footer [] [
-                text "Made by people who like to make nice things. © 2022"
+                text "Made by people who like to make nice things. © 2023"
```

Comment:
Interesting, I found that the © is also not required, should we remove it too?

Reply:
I think so, yeah 👍 

I also think we should add a comment that we intentionally have decided not to put copyright info in the footer, because that's the type of thing people will make unsolicited PRs to add 😄